pages/content/amp-dev/documentation/components/index.html

```diff
@@ -78,22 +106,86 @@ <h1>The AMP component catalogue</h1>
         </div>
       </div>
 
-      <p>If you want to know more about how AMP handles those under the hood read on with <a href="{{ g.doc('/content/amp-dev/about/how-amp-works.html', locale=doc.locale).url.path }}">How AMP works</a>.</p>
-
       <hr>
 
-      {% set categories = g.categories('/content/amp-dev/documentation/components/reference', locale=doc.locale) %}
-      {% include '/views/partials/category-filter.j2' %}
+      {% do doc.styles.addCssFile('css/components/molecules/teaser-component-code.css') %}
+      <section class="ap--section">
+        <h2>Highlighted Components</h2>
+        <div class="ap-m-teaser-component">
+          <div class="ap-m-teaser-container">
```

Comment:
Is there a reason we're not using the teaser partial here?

Reply:
Now using partial for the highlighted components